project/plugins.sbt

```diff
@@ -2,9 +2,8 @@ resolvers in Global += Resolver.sonatypeRepo("public")
 
 addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "1.7.1")
 addSbtPlugin("org.scalameta"      % "sbt-scalafmt"             % "2.4.3")
-addSbtPlugin("edu.gemini"         % "sbt-lucuma"               % "0.4.0")
+addSbtPlugin("edu.gemini"         % "sbt-lucuma"               % "0.4.2")
 addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.1.0")
 addSbtPlugin("com.eed3si9n"       % "sbt-buildinfo"            % "0.10.0")
 addSbtPlugin("com.typesafe.sbt"   % "sbt-git"                  % "1.0.2")
-addSbtPlugin("ch.epfl.scala"      % "sbt-scalafix"             % "0.9.31")
```

Comment:
Is this imported through sbt-lucuma now?

Reply:
Yes